Description: Debian-specific advice about recommended fonts-open-sans package
Forwarded: not-needed
Author: Nicholas Guriev <guriev-ns@ya.ru>
Date: Sat, 13 Feb 2021 08:06:10 +0300

--- a/Telegram/SourceFiles/platform/linux/specific_linux.cpp
+++ b/Telegram/SourceFiles/platform/linux/specific_linux.cpp
@@ -688,8 +688,9 @@ void start() {
 
 #ifdef DESKTOP_APP_USE_PACKAGED_FONTS
 	g_warning(
-		"Application was built without embedded fonts, "
-		"this may lead to font issues.");
+		"Application was built without embedded fonts, this may lead "
+		"to font issues. On Debian-based systems, make sure you have "
+		"the fonts-open-sans package installed");
 #endif // DESKTOP_APP_USE_PACKAGED_FONTS
 
 	// IBus has changed its socket path several times
